src/styles/index.scss

```diff
@@ -1,3 +1,80 @@
-body {
+@keyframes revolution-second-hand {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes revolution-minute-hand {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.stopwatch {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  &__object {
+    position: relative;
+    height: 80vmin;
+    width: 80vmin;
+    border: 1vmin dotted #000;
+    border-radius: 50%;
+  }
+
+  &__elements {
+    position: absolute;
+
+    &--minute-hand {
+      background-color: #0700ff;
+      height: 20vmin;
+      width: 3vmin;
+      bottom: 50%;
+      left: 48%;
+      transform-origin: bottom;
+      animation: revolution-minute-hand 3600s steps(60, end) infinite normal;
```

Comment:
The `steps()` function is used for the minute hand animation, which may not provide a smooth rotation. Consider using a linear timing function for a more realistic clock hand movement.